### ISSUE TITLE
Release request - simple rebuild

### ISSUE THREAD

**Issue by tgeens** [progrium/dockerhook#1]
Would it be possible to release a new build with updated dependencies ?

I can build this locally just fine, but some of your other projects (docker-plugins) are referencing the release here.

**Comment by tgeens** [comment]
The readme references some custom docker client build, but this seems to work just fine with the vanilla docker client ?

**Comment by josegonzalez** [comment]
Yeah the patch was probably old, but I'm going to try and get through updating this as I need it or something like it for Dokku.
